plugins/test/unit/plugins/importers/yum/parse/test_rpm.py

```diff
@@ -1,10 +1,16 @@
+from __future__ import absolute_import
+import os
+import rpm as rpm_module
```

Comment:
Presumably this should be in the second group of imports, since it is a 3rd-party package.